docker/go-1.10/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Go cross compiler (xgo): Go 1.10
```

Comment:
Please rename this folder to `go-1.10.0` to keep it consistent with the other images.